modules/nixos/services/bobaboard/default.nix

```diff
@@ -1,16 +1,92 @@
-{ lib, config, ... }:
+{ lib, config, inputs, system, pkgs, ... }:
 
 let
   cfg = config.services.bobaboard;
+  bobabackend-packages = inputs.boba-backend.packages."${system}";
 
   inherit (lib) mkEnableOption mkIf;
 in
 {
   options.services.bobaboard = {
     enable = mkEnableOption "BobaBoard";
   };
-
   config = mkIf cfg.enable {
     # @TODO(jakehamilton): Add configuration for BobaBoard here.
+
+    networking.firewall.allowedTCPPorts = [
+      4200
```

Comment:
This should be configurable with an option.

Reply:
Going to do options next PR :)